### DISCUSSION TITLE
Github desktop with bitbucket changing branch name not working

### DISCUSSION THREAD

**Discussion by ahmed-shehataa** [community/community#45839] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm facing a problem when changing branch name of local and remote repository of bitbucket.
If i have branch called X then change its name to Y and remove the remote branch X then push Y, the remote branch name still X not Y,
So how I can solve this issue?

**Comment by jge162** [comment]
`$ git branch -m X Y`
`$ git push origin --delete X`
`$ git push origin Y`

Use these git commands to update branch name locally and update the remote repository

Try and lmk

**Comment by djsunaysunay** [comment]
Ne istiyor sun Türkçe yazz
